### PR TITLE
batch migration for hyrax

### LINKED ISSUE #1404: Hyrax: verify and fix batch works
Find and fix the problems with creating and editing batches

### PR DESCRIPTION
**Pull request by jamesvanmil**
Fixes #1404

### CODE REVIEW

**Comment by 13miciunasl** [comment]
You have 5 rubocop failures

**Comment by 13miciunasl** [comment]
At least for me locally there are some label issues with the save work form

screen shot 2017-06-15 at 1 05 36 pm

**Comment by 13miciunasl** [comment]
And I also keep getting this error when hitting save
screen shot 2017-06-15 at 1 16 04 pm

**Comment by jamesvanmil** [comment]
I think those label issues are global - e.g. out of scope for my issue

**Comment by 13miciunasl** [comment]
This is what I got with trying to save images

screen shot 2017-06-15 at 3 47 23 pm

**Comment by 13miciunasl** [comment]
👍

File: app/views/_masthead.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<%= render '/shared/select_work_type_modal' if create_work_presenter.many? %>
```
Comment (hortongn) [inline app/views/_masthead.html.erb]: @jamesvanmil I'm just doing a code review of this PR.  Can you explain why you removed `if create_work_presenter.many?` from this view?  Both our develop branch and the Hyrax gem still have this conditional included.

Comment (jamesvanmil) [inline app/views/_masthead.html.erb]: `create_work_presenter` stopped responding to `#many?` on some view in my branch.  I figured our application would always have many, so it was safe to remove.

**Comment by jamesvanmil** [comment]
see https://github.com/uclibs/scholar_uc/pull/1476

**Pull request closed without merging**